src/Storage/Controllers/ProcessController.cs

```diff
@@ -83,7 +83,8 @@ public class ProcessController : ControllerBase
             string altinnTaskType = existingInstance.Process?.CurrentTask?.AltinnTaskType;
             string taskId = null;
 
-            if (processState?.CurrentTask?.FlowType != null && !processState.CurrentTask.FlowType.Equals("CompleteCurrentMoveToNext"))
+            var moveNextFlows = new[] { "CompleteCurrentMoveToNext", "AbandonCurrentMoveToNext" };
+            if (processState?.CurrentTask?.FlowType != null && !moveNextFlows.Contains(processState.CurrentTask.FlowType))
```

Comment:
Consider using `is not null` instead of `!= null`. The only difference (besides the syntax) is, that the compiler guarantees that no user-overloaded operator is called when using `is not null` instead of `!= nul`l (or `is null `instead of `== null`).

Reply:
Completely agree. I guess I took the "minimal PR changes" concept a step too far 🤷. I'll make the change now.